src/autoscaler/routes/routes.go

```diff
@@ -57,11 +57,15 @@ const (
 	PublicApiAggregatedMetricsHistoryPath      = "/{appId}/aggregated_metric_histories/{metricType}"
 	PublicApiAggregatedMetricsHistoryRouteName = "GetPublicApiAggregatedMetricsHistories"
 
-	PublicApiPolicyPath            = "/{appId}/policy"
+	PublicApiPolicyPath            = "/v1/apps/{appId:.+}/policy"
 	PublicApiGetPolicyRouteName    = "GetPolicy"
 	PublicApiAttachPolicyRouteName = "AttachPolicy"
 	PublicApiDetachPolicyRouteName = "DetachPolicy"
 
+	PublicApiCustomMetricsCredentialPath            = "/v1/apps/{appId:.+}/custom_metrics_credential"
+	PublicApiCreateCustomMetricsCredentialRouteName = "CreateCustomMetricsCredential"
+	PublicApiDeleteCustomMetricsCredentialRouteName = "DeleteCustomMetricsCredential"
+
```

Comment:
change `/v1/apps/{appId:.+}/custom_metrics_credential`  to `/v1/apps/{appId:.+}/credential`